__snapshots__/main.test.ts.snap

```diff
@@ -1251,7 +1251,7 @@ kind: Ingress
 metadata:
   annotations:
     kubernetes.io/ingress.class: public
-    nginx.ingress.kubernetes.io/proxy-body-size: 100m
+    nginx.ingress.kubernetes.io/proxy-body-size: 900m
```

Comment:
Why did it change?

Reply:
The models KLA uploading are bigger than 100m

Reply:
You've changes this manually?

Reply:
the pr that changing this value merged a few days ago here I just updating the tests